### PR TITLE
feat: Add Graph Connector sample app

### PR DESCRIPTION
**Pull request by formulahendry**
Demo:
graph-connector-app-demo

Architecture:
image

### CODE REVIEW

File: graph-connector-app/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Getting Started with Graph Connector Sample
+
+> Note: Please be advised that this sample repository is currently in **Public Preview**, with a lot of active development work taking place. Please expect breaking changes as we continue to iterate. 
+> 
+> We really appreciate your feedback! If you encounter any issue or error, please report issues to us following the [Supporting Guide](./../SUPPORT.md). Meanwhile you can make [recording](https://aka.ms/teamsfx-record) of your journey with our product, they really make the product better. Thank you!
+>  
+> This warning will be removed when the samples are ready for production.
+
+Graph Connector sample APP provides an easy way to use MS Graph client in TeamsFx to build a custom Graph Connector in Teams.
```
Comment (MuyangAmigo) [inline graph-connector-app/README.md:9]: This sample app showcases how to build custom Graph Connector with Azure Functions and query data using Microsoft Graph Client and TeamsFx SDK.

File: graph-connector-app/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+## Prerequisite
+- [NodeJS](https://nodejs.org/en/), fully tested on NodeJS 12, 14
+- An [Azure subscription](https://azure.microsoft.com/en-us/free/)
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+
+
+## What you will learn in this sample:
+- How to use TeamsFx to build frontend hosting on Azure for your tab app.
+- How to use TeamsFx to build backend hosting on Azure for your tab app.
+- How to use MS Graph client in TeamsFx to build a custom Graph Connector in Teams.
```
Comment (MuyangAmigo) [inline graph-connector-app/README.md:22]: How to use Microsoft Graph API to build a custom Graph Connector.

**Review by MuyangAmigo — APPROVED** [review]
Approve for README part.

File: graph-connector-app/.fx/configs/azure.parameters.dev.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "provisionParameters": {
+      "value": {
+        "resourceBaseName": "graphconnedev381195",
```
Comment (adashen) [inline graph-connector-app/.fx/configs/azure.parameters.dev.json:7]: The suffix should not be hard coded. Please refer to other samples.

File: graph-connector-app/.fx/configs/projectSettings.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+    "appName": "graphconnectorapp",
+    "projectId": "c88c8bc5-1e60-4a09-963d-02549a56322b",
```
Comment (adashen) [inline graph-connector-app/.fx/configs/projectSettings.json:3]: seems should not hard code projectId

File: graph-connector-app/.fx/configs/projectSettings.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    "version": "2.1.0",
+    "isFromSample": false,
```
Comment (adashen) [inline graph-connector-app/.fx/configs/projectSettings.json:5]: what is this value for?

File: graph-connector-app/api/package-lock.json [hunk 6]
```diff
@@ -0,0 +1,1562 @@
+{
```
Comment (adashen) [inline graph-connector-app/api/package-lock.json:1]: Do we need to submit package-lock?

File: graph-connector-app/tabs/package.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
+{
+    "name": "teamsfx-template-tab",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@fluentui/react-northstar": "^0.54.0",
+        "@microsoft/mgt-element": "2.4.1-next.teamsfx.3511fba",
```
Comment (adashen) [inline graph-connector-app/tabs/package.json:7]: is 7-9 necessary? Seems these are for graph toolkit. Do we use it in this sample?

**Comment by formulahendry** [comment]
Close this PR. Will create a new one targeting GA branch

**Pull request closed without merging**